Content.Server/GameObjects/Components/Power/WirePlacerComponent.cs

```diff
@@ -44,9 +45,13 @@ public void AfterAttack(AfterAttackEventArgs eventArgs)
             if (found)
                 return;
 
+            bool hasItemSpriteComp = Owner.TryGetComponent(out SpriteComponent itemSpriteComp);
+
+            if (Owner.TryGetComponent(out StackComponent stack) && !stack.Use(1))
```

Comment:
This doesn't actually delete the stack when it's empty. StackComponent could definitely use some improvement in that regard.

Reply:
what's this then
https://github.com/space-wizards/space-station-14/blob/1996893a262bed6290def614b0d4a000f9b5ab78/Content.Server/GameObjects/Components/Stack/StackComponent.cs#L37-L40

Reply:
I feel stupid now.